.idea/misc.xml

```diff
@@ -24,7 +24,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
huh?!

Reply:
Yeah, idk. It keeps changing in my desk environment

Reply:
Very strange problem. What version of Android Studio are you using? It may also be an issue with the .gitignore and .idea folder.

https://stackoverflow.com/questions/32878888/android-idea-misc-xmls-languagelevel-tag-keeps-changing-jdks